tools/heroku/copy-env-vars.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Source: http://blog.nonuby.com/blog/2012/07/05/copying-env-vars-from-one-heroku-app-to-another/
+
+set -e
+
+sourceApp="$1"
+targetApp="$2"
+defaultKeys=(APP_URI APP_VERSION DATABASE_URL FOO_COOKIE_SECRET NEW_RELIC_LICENSE_KEY NEW_RELIC_LOG PAPERTRAIL_API_TOKEN PG_MAX_CONNECTIONS REDISCLOUD_URL)
```

Comment:
Should this be updated: NEW_RELIC_LICENSE_KEY, NEW_RELIC_LOG or are they in use

Reply:
they are used in production